addons/Dockerfile.dockerignore

```diff
@@ -0,0 +1,2 @@
+artifacts
```

Comment:
Just a single `.dockerignore` in repo root is enough (note the file needs to be named just `.dockerignore`). It's not necessary to have it in all these directories since all the `docker-build` targets `cd` to repo root for passing context to Docker.

nit: add a trailing `/` to these entries indicate they are directories.

Reply:
I didn't know about `Dockerfile.dockerignore`. Thanks for the explanation!